doc/api/util.md

```diff
@@ -65,7 +65,9 @@ is wrapped in an `Error` with the original value stored in a field named
   callbackFunction((err, ret) => {
     // When the Promise was rejected with `null` it is wrapped with an Error and
     // the original value is stored in `reason`.
-    err && err.hasOwnProperty('reason') && err.reason === null;  // true
+    console.log(err && Object.prototype.hasOwnProperty.call(err, 'reason') &&
```

Comment:
as long as this is being refactored can you pull hasOwnProperty off at the top in case user code modifies it

Reply:
I went for removing as it is not necessary in the example. It is strictly testing for `null` and that has to be explicitly set.